oci-pipeline-examples/oci-java-micronaut-devops-example/gradle.properties

```diff
@@ -0,0 +1 @@
+micronautVersion=1.3.4
```

Comment:
You might mention that the user should check for the proper (latest) version to use here?

Reply:
Done ,thank you.